_entities.md

```diff
@@ -145,6 +145,32 @@ bid      | The current bid price, or the price we quote when buying the asset.
 currency | The currency that is used in the `ask` and `bid` prices.
 pair     | The currency pair AB represents moving from A to B.
 
+## Phone Object
+
+> An example phone encoded in JSON looks like this:
+
+```
+{
+  "e164Masked": "+XXXXXXXXX04",
+  "id": "1d78aeb5-43ac-4ee8-8d28-1291b5d8355c",
+  "internationalMasked": "+X XXX-XXX-XX04",
+  "nationalMasked": "(XXX) XXX-XX04",
+  "primary": true,
+  "verified": true
+}
+```
+
+A phone number is required for registration and, when enabled, provides additional security to the user's account through the use of Two-Factor Authentication (2FA).
+
+Property            | Description
+------------------- | ---------------------------------------------------------------------------------------------------------
+e164Masked          | The masked representation of the phone number in the [E.164 format](https://en.wikipedia.org/wiki/E.164).
+id                  | A unique ID in the Uphold platform identifying the phone.
+internationalMasked | The masked representation of the phone number in international format.
```

Comment:
IMO it would he helpful to provide a link describing exactly what we mean by "international format". I was thinking about one of these:

- https://github.com/google/libphonenumber/blob/master/javascript/i18n/phonenumbers/phonenumberutil.js#L885-L894
- https://javadoc.io/doc/com.googlecode.libphonenumber/libphonenumber/latest/com/google/i18n/phonenumbers/PhoneNumberUtil.PhoneNumberFormat.html
- https://github.com/ruimarinho/google-libphonenumber/blob/v3.2.1/src/phonenumberutil.js#L886-L895

WDYT?

Reply:
Update: After syncing with @nunofgs I've added the third link to the docs.